testing/scenario_app/lib/src/texture.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:ui';
+
+import 'package:scenario_app/src/scenarios.dart';
+
+import 'channel_util.dart';
+import 'scenario.dart';
+
+/// Displays a platform texture with the given width and height.
+class DisplayTexture extends Scenario {
+  /// Creates the DisplayTexture scenario.
+  ///
+  /// The [dispatcher] parameter must not be null.
```

Comment:
our style docs mentions that it's preferred to avoid comments that can otherwise be communicated through the type system or the method signature.

Reply:
Omitting the comment fails a lint check, so I don't think this can be removed. This matches whats done in other test scenarios.